client/components/formatted-header/style.scss

```diff
@@ -36,3 +36,20 @@
 		padding: 0 20px 24px;
 	}
 }
+
+// Responsive header
+.formatted-header.is-responsive {
```

Comment:
Nit: does it make sense the following code style instead? No hard preference, just providing an alternative.

```
.formatted-header {
	&.is-responsive {
		...
	}
}
```

Reply:
I didn't want to create another nesting level here. There's already a media query and child selectors inside so we're getting pretty deep in terms of nesting already (3 levels while the recommendation it to have 2 levels at most).

Reply:
That's fair, let's keep it as-is then 👍